owncloudApp/src/main/java/com/owncloud/android/ui/activity/ToolbarActivity.kt

```diff
@@ -137,4 +145,21 @@ abstract class ToolbarActivity : BaseActivity() {
     private fun getRootToolbar(): ConstraintLayout = findViewById(R.id.root_toolbar)
 
     private fun getStandardToolbar(): Toolbar = findViewById(R.id.standard_toolbar)
+
+    override fun onPrepareOptionsMenu(menu: Menu): Boolean {
```

Comment:
This should be done better in `onCreateOptionsMenu` or you will have problems with classes that extend this one and use that method

Reply:
Changed to `onCreateOptionsMenu`